README.md

```diff
@@ -309,6 +309,18 @@ git = "https://github.com/japaric/steed"
 stage = 2
 ```
 
+## Cargo subcommand
```

Comment:
I don't entire understand the title here. What about something like "Check-only libstd build"?

Reply:
This was left over from the original incarnation of this PR. I'll change it to something more suitable.